swift-sdk/IterableMessaging.swift

```diff
@@ -205,6 +211,9 @@ public extension Notification.Name {
 
 /// Encapsulates Inbox Session
 @objcMembers public final class IterableInboxSession: NSObject, Codable {
```

Comment:
Not needed for this PR.... Does this class need to be public?

Reply:
I also added this to be looked at with the evaluation ticket